edx_proctoring/templates/emails/proctoring_attempt_submitted_email.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+
+<p>
+    {% blocktrans %}
+        Hi {username},
+    {% endblocktrans %}
+</p>
+<p>
+    {% blocktrans %}
+        Your proctored exam {{ exam_name }} in
+        <a href="{{ course_url }}">{{ course_name }}</a> was submitted
+        successfully and will now be reviewed to ensure all proctoring exam
+        rules were followed. You will receive an email with your updated exam
```

Comment:
@marcotuts @andy-armstrong Re the promise of an email with exam results, do you feel we need softer language? "You should receive an email within 5 business days" vs "You will receive an email..."?